azure-mgmt-appservice/src/main/java/com/microsoft/azure/management/appservice/implementation/AppServiceManager.java

```diff
@@ -202,4 +201,105 @@ public FunctionApps functionApps() {
         }
         return functionApps;
     }
+
+    /**
+     *
+     * @return Returns the latest Windows runtime stacks
+     */
+    public Runtimes latestWindowsRuntimes() {
+        Hashtable<String, RuntimeVersion> variousRuntimes = new Hashtable<>();
+        variousRuntimes.put(JavaVersion.ComponentName.toLowerCase(), JavaVersion.OFF);
+        variousRuntimes.put(NodeVersion.ComponentName.toLowerCase(), NodeVersion.OFF);
+        variousRuntimes.put(WebContainer.ComponentName.toLowerCase(), WebContainer.OFF);
+        variousRuntimes.put(PythonVersion.ComponentName.toLowerCase(), PythonVersion.OFF);
+        variousRuntimes.put(PhpVersion.ComponentName.toLowerCase(), PhpVersion.OFF);
+        variousRuntimes.put(NetFrameworkVersion.ComponentName.toLowerCase(), NetFrameworkVersion.OFF);
+
+        Iterator<ApplicationStackInner> stackIter = this.innerManagementClient.providers().getAvailableStacks("Windows").iterator();
+        while (stackIter.hasNext()) {
```

Comment:
PagedList uses raw iterator since it implements List interface. It is required to work with iterator directly when we implement list interface but I thought consumption part like this can keep simple by taking advantage of `for construct` abstraction for itr handling.